### ISSUE TITLE
Failing to retrieve UniProt data

### ISSUE THREAD

**Issue by HobnobMancer** [HobnobMancer/cazy_webscraper#100]
## Describe the bug
When using `cw_get_uniprot_data` to retrieve data from UniProt, no data is retrieved and added to the local CAZyme database

## To Reproduce
1. Build a local CAZyme database: `cazy_webscraper <email> -o cazy.db`
2. `cw_get_uniprot_data cazy.db --families 20 --pdb`
```bash
Built output directory: .cazy_webscraper_2022-11-18_20-03-08/uniprot_data_retrieval
Using default CAZy class synonyms
Retrieving GenBank accessions for selected CAZy classes: 0it [00:00, ?it/s]
Applying CAZy family filter(s)
Retrieving GenBank accessions for selected CAZy families:   0%|                                                           | 0/1 [00:00<?, ?it/s]Retrieving CAZymes for CAZy family PL20
Retrieving GenBank accessions for selected CAZy families: 100%|███████████████████████████████████████████████████| 1/1 [00:00<00:00, 20.02it/s]
Applying no taxonomic filters
Retrieving UniProt data for 76
Batch retrieving UniProt IDs: 11it [00:00, 15.03it/s]                                                                                           
Batch retrieving protein data from UniProt: 0it [00:00, ?it/s]
Adding data to the local CAZyme database
Retrieving existing UniProt records from db: 0it [00:00, ?it/s]
Separating new and existing records: 0it [00:00, ?it/s]
Loading existing PDB db records: 0it [00:00, ?it/s]
Identifying new PDBs to add to db: 0it [00:00, ?it/s]
Loading existing Genbank_Pdbs db records: 0it [00:00, ?it/s]
Identifying new protein-PDB relationships to add to db: 0it [00:00, ?it/s]
```
No data is retrieved from UniProt.

## Expected behavior
Retrieve data from UniProt and add to the local CAZyme database

**Comment by mherold1** [comment]
Hi,
when testing the tool I noticed that I had problems at this step and after searching I noticed that the requests via:
get_uniprot_accession() from https://github.com/HobnobMancer/saintBioutils/blob/master/saintBioutils/uniprot/__init__.py were failing. 
Apparently the UniProt API has recently changed.
Maybe this is helpful for replacing the queries: https://github.com/multimeric/Unipressed

**Comment by mherold1** [comment]
Thanks for the quick response.
Are you sure that the issue is related to the bioservices version? I had 1.10.4 (the latest?) installed.
I went through the script:
https://github.com/HobnobMancer/cazy_webscraper/blob/master/cazy_webscraper/expand/uniprot/get_uniprot_data.py
and where it is failing for me is at the EMBL to Uniprot accessions mapping step through `saintBioutils`. The returned `uniprot_gkb_dict` is empty in:
https://github.com/HobnobMancer/cazy_webscraper/blob/master/cazy_webscraper/expand/uniprot/get_uniprot_data.py#L180
When testing the other script it was failing at the request (or L94):
https://github.com/HobnobMancer/saintBioutils/blob/master/saintBioutils/uniprot/__init__.py#L98

**Comment by HobnobMancer** [comment]
This issues should now be fixed in v2.2.3 - PR #103
